api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractModel.java

```diff
@@ -0,0 +1,52 @@
+package com.nicknathanjustin.streamercontracts.contracts;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Entity
+@Table(name = "contracts")
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@Setter(value = AccessLevel.PACKAGE)
+@Getter
+public class ContractModel {
```

Comment:
Would we add the paypalPaymentId to this class or does that go someplace else? Maybe a paymentsModel makes sense? Im not sure where the place to stick that is yet, maybe we could discuss that on wednesday

Reply:
We should add it to the donation table. I'll make the change